dpnp/backend_mathematical.pyx

```diff
@@ -112,6 +113,27 @@ func_map[add_name][int32_name][float32_name] = [float64_name, & custom_elemwise_
 func_map[add_name][int32_name][int64_name] = [int64_name, & custom_elemwise_add_c[int, long, long]]
 func_map[add_name][int32_name][int32_name] = [int32_name, & custom_elemwise_add_c[int, int, int]]
 
+
+cpdef dparray dpnp_absolute(dparray x):
+    result = dparray(x.shape, dtype=x.dtype)
+
+    cdef dparray_shape_type shape_x = x.shape
+    cdef size_t dim_x = x.ndim
+
+    if dim_x > 2:
+        raise NotImplementedError
+
+    if dim_x == 2:
+        for i in range(shape_x[0]):
+            for j in range(shape_x[1]):
+                result[i, j] = x[i, j] if x[i, j] >= 0 else -1*x[i, j]
```

Comment:
```suggestion
                elem = x[i, j]
                result[i, j] = elem if elem >= 0 else -elem
```
rhetorical question - how many time in original code the operator `[]` (__get_item__) called?

Reply:
Also, it looks like all elements are involved into this.
So, it is simpler to use something like:
```
for i in range(x.size):
    result.item(i) = x.item(i)
```